WooCommerce/Classes/ViewRelated/Products/Edit Product/Shipping Settings/ProductShippingSettingsViewController.swift

```diff
@@ -46,6 +43,21 @@ final class ProductShippingSettingsViewController: UIViewController {
         self.shippingSettingsService = shippingSettingsService
         self.onCompletion = completion
 
+        // TODO-2580: re-enable shipping class for `ProductVariation` when the API issue is fixed.
+        switch product {
+        case is Product:
+            sections = [
+                Section(rows: [.weight, .length, .width, .height]),
+                Section(rows: [.shippingClass])
+            ]
+        case is ProductVariation:
+            sections = [
+                Section(rows: [.weight, .length, .width, .height])
+            ]
```

Comment:
Should we extract this into a testable helper/function? or do you think it's not necessary?

Reply:
Similar to [this comment](https://github.com/woocommerce/woocommerce-ios/pull/2588#discussion_r464170659), I'll make sure to extract the view logic to a view model to have unit tests